Menu/GameShell/10_Settings/Wifi/wifi_list.py

```diff
@@ -159,9 +159,9 @@ def TryDisconnect(self):
     def OnLoadCb(self):
         if self._Wireless.GetCurrentNetworkID(self._Wireless.GetIwconfig()) == self._NetworkId \
            and self._Wireless.GetWirelessIP('') is not None:
-            self._FootMsg[1] = "Disconnect"
+            self._FootMsg[2] = "Disconnect"
```

Comment:
You added an element to the list but you forgot to change the index here, so the diconnect button showed twice on the footer. :(